### PR TITLE
ascanrules: Promotions

### PR DESCRIPTION
**Pull request by kingthorin**
- Moved scan rule and unit test classes.
- Relocated help and Messages.properties content.
- Updated CHANGELOGs.

Signed-off-by: kingthorin <kingthorin@users.noreply.github.com>

### CODE REVIEW

**Comment by thc202** [comment]
Better release the add-ons before starting with the promotions.

**Comment by kingthorin** [comment]
OH oops yes we had talked about that. My bad.

**Comment by kingthorin** [comment]
Rebased & CHANGELOGs tweaked.

**Comment by kingthorin** [comment]
Rebased & CHANGELOGs tweaked again.

**Comment by kingthorin** [comment]
Ping, can I get some reviews on this so that I can move along with others?

**Comment by thc202** [comment]
We shouldn't merge until we target 2.12 IMO.

**Comment by kingthorin** [comment]
Shall I do the Beta ➡️ Release set as another commit here then?

**Comment by ricekot** [comment]
Worth changing to WIP until it's ready to be merged?

**Comment by psiinon** [comment]
Arg. I've pushed the beta -> release promotions but it looks like it pulled all of the other commits as well.
Guess I need to rebase?

**Comment by psiinon** [comment]
Hopefully ready for review now 😄

**Comment by psiinon** [comment]
No, I've got some tidying up still to do...

**Comment by psiinon** [comment]
Think thats it...

**Comment by psiinon** [comment]
Missed committing one of the test files, hopefully will work this time..

**Comment by psiinon** [comment]
Hopefully fixed the conflicts without breaking anything...

**Comment by psiinon** [comment]
All checks passed 😁

**Comment by kingthorin** [comment]
I’ll tackle this in the next few hours, it’s too much for a cell screen 😉

**Comment by thc202** [comment]
Thank you both!

File: addOns/ascanrulesBeta/src/main/java/org/zaproxy/zap/extension/ascanrulesBeta/SourceCodeDisclosureFileInclusionScanRule.java
Comment (kingthorin) [inline addOns/ascanrulesBeta/src/main/java/org/zaproxy/zap/extension/ascanrulesBeta/SourceCodeDisclosureFileInclusionScanRule.java:281]: Is there a changelog entry that should have accompanied this change?

**Comment by kingthorin** [comment]
Fixup or do that at merge?

**Comment by psiinon** [comment]
At merge :)

**Comment by kingthorin** [comment]
Hmmm GitHub is still slowing this open for me, weird.

**Comment by kingthorin** [comment]
Hmmm I can’t approve for some reason.
E5944638-D399-4328-97B7-9F2DFB3E0290

**Comment by thc202** [comment]
You are the author of the PR.

**Comment by kingthorin** [comment]
Omg, yup that’s how my week is going….

**Review by psiinon — APPROVED** [review]

**Pull request merged**
